includes/TaskManagers/TaskManager.php

```diff
@@ -7,6 +7,7 @@ final class TaskManager {
 
 	 protected $task_managers = array(
 		 'NewfoldLabs\\WP\Module\\Onboarding\\TaskManagers\\PluginInstallTaskManager',
+		 'NewfoldLabs\\WP\Module\\Onboarding\\TaskManagers\\PluginUninstallTaskManager',
```

Comment:
Let's combine these `TaskManagers` into a single `PluginsManager` so that we are not creating a new task manager for every single operation. Long term the goal is to simply remove all these and move to a single `TaskManager`